perf/data.ts

```diff
@@ -50,13 +50,15 @@ export function randomASCIIString(len: number): string {
   return randomStringInternal(len, 128);
 }
 
+// Shared array buffer to reduce allocations.
+const codes: number[] = [];
```

Comment:
In my measurements this function was taking ~30% of execution time on the 16MB benchmark, mostly due to GC. Sharing this buffer reduced the time to ~15%.